openadapt/utils.py

```diff
@@ -946,6 +948,23 @@ def get_posthog_instance() -> DistinctIDPosthog:
     return posthog
 
 
+def upload_file_to_s3(file_path: str, body: dict) -> requests.Response:
+    """Upload a file to an S3 bucket.
+
+    Args:
+        file_path (str): The path to the file to upload.
+        body (dict): The body of the request.
+    """
+    filename = os.path.basename(file_path)
```

Comment:
```suggestion
    file_name = os.path.basename(file_path)
```

Reply:
What do you think about logging this path?

Reply:
Done